README.md

```diff
@@ -18,9 +18,18 @@ npm install eslint-teamcity --save-dev
 ```
 
 ## Usage
+* as onrun eslint formatter:
 ```sh
   eslint --format './node_modules/eslint-teamcity/index.js' myfiletolint.js
 ```
+* as post runnig formatter, work with standart eslint json report
```

Comment:
Ok cool! The ESLint docs weren't particularly clear on how to do this. Could you include this example in the README please? I think it would be helpful